include/esp/lwesp_ll_buddy.h

```diff
@@ -68,7 +68,7 @@ extern espr_t esp_flash_initialize();
 // be copied from the RX buffer to ESP mem before the buffer overflows.
 #define RX_BUFFER_LEN 0x1000
 #if !defined(ESP_MEM_SIZE)
-    #define ESP_MEM_SIZE 0x1000
+    #define ESP_MEM_SIZE 0x2000
```

Comment:
Is that the size of the ESP's heap? Can we afford another 4k of RAM to be given to ESP? @DRracer :question:

Reply:
Separated this into a commit for easier revert.